packages/integrations/solid/README.md

```diff
@@ -44,12 +44,13 @@ Now, apply this integration to your `astro.config.*` file using the `integration
 __`astro.config.mjs`__
 
 ```js
+import { defineConfig } from 'astro/config';
 import solid from '@astrojs/solid-js';
 
-export default {
-  // ...
-  integrations: [solid()],
-}
+export default defineConfig({
+   // ...
+   integrations: [solid()],
+});
```

Comment:
Looks like there are other readmes like svelte, turbolinks, react, lit, image, etc that could use this update too to be consistent. Also maybe we should keep the indent two spaces instead of three.

Reply:
Agree with @bluwy — a good change to make across all READMEs using the outdated config style. And +1 on indenting by two spaces.

Reply:
I just went through and updated all of these!